### ISSUE TITLE
Installing Optimint on a new cosmos-sdk application that uses Ignite to generate the chain shows some exit logs

### ISSUE THREAD

**Issue by YazzyYaz** [celestiaorg/rollmint#548]
When running the following instructions on a new Cosmos-SDK chain being generated with ignite scaffolding ([per docs](https://docs.celestia.org/developers/scaffold-wordle#scaffolding-the-wordle-chain)), installing Optimint steps generate the following output:

```
go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/celestiaorg/cosmos-sdk@v0.45.4-optimint-v0.3.5
go mod tidy
go mod download
```

```
go: downloading github.com/tendermint/tendermint v0.35.9
go: finding module for package github.com/tendermint/tendermint/proxy
go: finding module for package github.com/tendermint/tendermint/types/time
go: finding module for package github.com/cosmos/cosmos-sdk/x/bank/testutil
go: finding module for package github.com/tendermint/tendermint/crypto/armor
go: finding module for package github.com/tendermint/tendermint/state
go: finding module for package github.com/cosmos/cosmos-sdk/x/auth/migrations/legac
ytx
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/gov/types/v1: module github.com/cosmos/cosmo
s-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45.4-
optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/gov/t
ypes/v1
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1: module github.com/cosmos/
cosmos-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.
45.4-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/
gov/types/v1beta1
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/pruning/types: module github.com/cosmos/cosmos
-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45.4-o
ptimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/pruning/
types
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/client imports
        github.com/tendermint/tendermint/mempool: module github.com/tendermint/tend
ermint@latest found (v0.35.9), but does not contain package github.com/tendermint/t
endermint/mempool
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/client/grpc/tmservice imports
        github.com/tendermint/tendermint/rpc/core/types: module github.com/tendermi
nt/tendermint@latest found (v0.35.9), but does not contain package github.com/tende
rmint/tendermint/rpc/core/types
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/genutil imports
        github.com/tendermint/tendermint/p2p: module github.com/tendermint/tendermi
nt@latest found (v0.35.9), but does not contain package github.com/tendermint/tende
rmint/p2p
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/group/keeper imports
github.com/YazzyYaz/wordle/app imports                                      [7/210]
        github.com/cosmos/cosmos-sdk/x/group/keeper imports
        github.com/cosmos/cosmos-sdk/x/auth/middleware: module github.com/cosmos/co
smos-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45
.4-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/au
th/middleware
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/testutil/network imports
        github.com/tendermint/tendermint/libs/cli/flags: module github.com/tendermi
nt/tendermint@latest found (v0.35.9), but does not contain package github.com/tende
rmint/tendermint/libs/cli/flags
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/testutil/network imports
        github.com/tendermint/tendermint/proxy: module github.com/tendermint/tender
mint@latest found (v0.35.9), but does not contain package github.com/tendermint/ten
dermint/proxy
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/testutil/network imports
        github.com/tendermint/tendermint/types/time: module github.com/tendermint/t
endermint@latest found (v0.35.9), but does not contain package github.com/tendermin
t/tendermint/types/time
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/group/keeper tested by
        github.com/cosmos/cosmos-sdk/x/group/keeper.test imports
        github.com/cosmos/cosmos-sdk/x/bank/testutil: module github.com/cosmos/cosm
os-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45.4
-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/bank
/testutil
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/crypto/keyring imports
        github.com/cosmos/cosmos-sdk/crypto imports
        github.com/tendermint/tendermint/crypto/armor: module github.com/tendermint
/tendermint@latest found (v0.35.9), but does not contain package github.com/tenderm
int/tendermint/crypto/armor
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/host import
s
        github.com/cosmos/ibc-go/v5/modules/core/04-channel/types tested by
        github.com/cosmos/ibc-go/v5/modules/core/04-channel/types.test imports
        github.com/tendermint/tendermint/state: module github.com/tendermint/tender
mint@latest found (v0.35.9), but does not contain package github.com/tendermint/ten
dermint/state
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts tested by
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts.test import
s
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/group/keeper tested by
        github.com/cosmos/cosmos-sdk/x/group/keeper.test imports
        github.com/cosmos/cosmos-sdk/x/bank/testutil: module github.com/cosmos/cosm
os-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45.4
-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/bank
/testutil
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/crypto/keyring imports
        github.com/cosmos/cosmos-sdk/crypto imports
        github.com/tendermint/tendermint/crypto/armor: module github.com/tendermint
/tendermint@latest found (v0.35.9), but does not contain package github.com/tenderm
int/tendermint/crypto/armor
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/host import
s
        github.com/cosmos/ibc-go/v5/modules/core/04-channel/types tested by
        github.com/cosmos/ibc-go/v5/modules/core/04-channel/types.test imports
        github.com/tendermint/tendermint/state: module github.com/tendermint/tender
mint@latest found (v0.35.9), but does not contain package github.com/tendermint/ten
dermint/state
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts tested by
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts.test import
s
        github.com/cosmos/ibc-go/v5/testing/simapp imports
        github.com/cosmos/ibc-go/v5/testing/simapp/params imports
        github.com/cosmos/cosmos-sdk/x/auth/migrations/legacytx: module github.com/
cosmos/cosmos-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-
sdk@v0.45.4-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos
-sdk/x/auth/migrations/legacytx
```

Then running this shows an exit status:

```
root@yaz-workshop:~/wordle# ignite scaffold module wordle --dep bank
? Your saved project changes have not been committed. To enable reverting to your c
urrent state, commit your saved changes. Do you want to proceed with scaffolding wi
thout committing your saved changes Yes
◥ Scaffolding... go: finding module for package github.com/cosmos/cosmos-sdk/x/grou
p/keeper
go: finding module for package github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1
go: finding module for package github.com/cosmos/cosmos-sdk/x/gov/types/v1
go: finding module for package github.com/cosmos/cosmos-sdk/x/group
◥ Scaffolding... go: finding module for package github.com/cosmos/cosmos-sdk/x/grou
p/module
go: finding module for package github.com/cosmos/cosmos-sdk/pruning/types
go: finding module for package github.com/cosmos/cosmos-sdk/x/auth/migrations/legac
ytx
◣ Scaffolding... go: found github.com/cosmos/cosmos-sdk/x/group in github.com/cosmo
s/cosmos-sdk/x/group v0.0.0-20220119104752-a036c530fc0c
go: found github.com/cosmos/cosmos-sdk/x/group/keeper in github.com/cosmos/cosmos-s
dk/x/group v0.0.0-20220119104752-a036c530fc0c
go: found github.com/cosmos/cosmos-sdk/x/group/module in github.com/cosmos/cosmos-s
dk/x/group v0.0.0-20220119104752-a036c530fc0c
◥ Scaffolding... go: finding module for package github.com/tendermint/tendermint/me
mpool
go: finding module for package github.com/cosmos/cosmos-sdk/pruning/types
go: finding module for package github.com/cosmos/cosmos-sdk/x/gov/types/v1
go: finding module for package github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1
go: finding module for package github.com/tendermint/tendermint/rpc/core/types
go: finding module for package github.com/tendermint/tendermint/p2p
go: finding module for package github.com/cosmos/cosmos-sdk/x/auth/middleware
go: finding module for package github.com/tendermint/tendermint/libs/cli/flags
◤ Scaffolding... go: finding module for package github.com/tendermint/tendermint/pr
oxy
go: finding module for package github.com/cosmos/cosmos-sdk/x/bank/testutil
go: finding module for package github.com/tendermint/tendermint/types/time
go: finding module for package github.com/tendermint/tendermint/crypto/armor
go: finding module for package github.com/tendermint/tendermint/state
go: finding module for package github.com/cosmos/cosmos-sdk/x/auth/migrations/legac
ytx
◤ Scaffolding... github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/gov/types/v1: module github.com/cosmos/cosmo
s-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45.4-
optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/gov/t
ypes/v1
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1: module github.com/cosmos/
cosmos-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.
cosmos-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.
45.4-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/
gov/types/v1beta1
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/pruning/types: module github.com/cosmos/cosmos
-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45.4-o
ptimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/pruning/
types
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/client imports
        github.com/tendermint/tendermint/mempool: module github.com/tendermint/tend
ermint@latest found (v0.35.9), but does not contain package github.com/tendermint/t
endermint/mempool
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/client/grpc/tmservice imports
        github.com/tendermint/tendermint/rpc/core/types: module github.com/tendermi
nt/tendermint@latest found (v0.35.9), but does not contain package github.com/tende
rmint/tendermint/rpc/core/types
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/genutil imports
        github.com/tendermint/tendermint/p2p: module github.com/tendermint/tendermi
nt@latest found (v0.35.9), but does not contain package github.com/tendermint/tende
rmint/p2p
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/group/keeper imports
        github.com/cosmos/cosmos-sdk/x/auth/middleware: module github.com/cosmos/co
smos-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45
.4-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/au
th/middleware
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/testutil/network imports
        github.com/tendermint/tendermint/libs/cli/flags: module github.com/tendermi
nt/tendermint@latest found (v0.35.9), but does not contain package github.com/tende
rmint/tendermint/libs/cli/flags
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/testutil/network imports
        github.com/tendermint/tendermint/proxy: module github.com/tendermint/tender
mint@latest found (v0.35.9), but does not contain package github.com/tendermint/ten
dermint/proxy
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/testutil/network imports
        github.com/tendermint/tendermint/types/time: module github.com/tendermint/t
endermint@latest found (v0.35.9), but does not contain package github.com/tendermin
t/tendermint/types/time
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/cosmos-sdk/x/group/keeper tested by
        github.com/cosmos/cosmos-sdk/x/group/keeper.test imports
        github.com/cosmos/cosmos-sdk/x/bank/testutil: module github.com/cosmos/cosm
os-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-sdk@v0.45.4
-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos-sdk/x/bank
/testutil
github.com/YazzyYaz/wordle/testutil/network imports
        github.com/cosmos/cosmos-sdk/crypto/keyring imports
        github.com/cosmos/cosmos-sdk/crypto imports
        github.com/tendermint/tendermint/crypto/armor: module github.com/tendermint
/tendermint@latest found (v0.35.9), but does not contain package github.com/tenderm
int/tendermint/crypto/armor
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/host import
s
        github.com/cosmos/ibc-go/v5/modules/core/04-channel/types tested by
        github.com/cosmos/ibc-go/v5/modules/core/04-channel/types.test imports
        github.com/tendermint/tendermint/state: module github.com/tendermint/tender
mint@latest found (v0.35.9), but does not contain package github.com/tendermint/ten
dermint/state
github.com/YazzyYaz/wordle/app imports
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts tested by
        github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts.test import
s
        github.com/cosmos/ibc-go/v5/testing/simapp imports
        github.com/cosmos/ibc-go/v5/testing/simapp/params imports
        github.com/cosmos/cosmos-sdk/x/auth/migrations/legacytx: module github.com/
cosmos/cosmos-sdk@latest found (v0.46.2, replaced by github.com/celestiaorg/cosmos-
sdk@v0.45.4-optimint-v0.3.5), but does not contain package github.com/cosmos/cosmos
-sdk/x/auth/migrations/legacytx
exit status 1
```

**Comment by tzdybal** [comment]
Latest Ignite CLI expects latest cosmos-sdk (v0.46.x branch). It should be ready today. Currently rollmint is at v0.46.1 compatibility, so I need to rebase for v0.46.2, and I need to release appropriate cosmos-sdk-rollmint version.

**Comment by tzdybal** [comment]
This will be resolved when #528 is done.

**Comment by tzdybal** [comment]
Closing, as v0.4.0 solves the issue.
